tests/lib/rules/no-todo.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @fileoverview Forbid the use of QUnit.todo
+ * @author Baptiste Doucerain
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-todo"),
+    RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("no-todo", rule, {
+    valid: [
+        "QUnit.module.test('Name', function() { });",
+        "QUnit.test('Name', function() { });",
+        "module.test('Name', function() { });",
+        "test('Name', function() { });"
```

Comment:
Valid test cases that need to be added:

* `foo.todo('Name', function() { });`
* `foo.bar.todo('Name', function() { });`
* `module.foo.todo('Name', function() { });`
* `test.foo.todo('Name', function() { });`